packaging/templates/config/amoy/config.toml

```diff
@@ -458,6 +458,28 @@ peer_query_maj23_sleep_duration = "200ms"
 # reindex events in the command-line tool.
 discard_abci_responses = false
 
+# If set to true, CometBFT will force compaction to happen for databases that support this feature.
+# and save on storage space. Setting this to true is most benefits when used in combination
+# with pruning as it will phyisically delete the entries marked for deletion.
+# false by default (forcing compaction is disabled).
+compact = false
+
+# To avoid forcing compaction every time, this parameter instructs CometBFT to wait 
+# the given amount of blocks to be pruned before triggering compaction.
+# It should be tuned depending on the number of items. If your retain height is 1 block,
+# it is too much of an overhead to try compaction every block. But it should also not be a very
+# large multiple of your retain height as it might occur bigger overheads.
+compaction_interval = 1000
+
+[storage.pruning]
+
+# The time period between automated background pruning operations.
+interval = "10m0s"
+
+# Indexer pruning enabling. Note that this has not been tested in production
+# It should be enabled on full nodes to confirm it is not impacting performance. 
+indexer_pruning_enabled = false
```

Comment:
Should we mention this hasn't been tested? I guess it comes from comet.
Since we are testing this on amoy, the misleading comment should be removed imho